index.html

```diff
@@ -5,11 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <link rel="stylesheet" href="/teras-mavr-cv/assets/css/reset.css">
-
     <title>Document</title>
+
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/reset.css">
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500;700&display=swap" rel="stylesheet">
+
 </head>
 <body>
+  <!-- main_section -->
+  <div class="main_section">
```

Comment:
[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) why not ```main```?